### ISSUE TITLE
[QA] richdocuments app fails in 10.14.0

### ISSUE THREAD

**Issue by jnweiger** [owncloud/core#41186]
That is a regression.
Seen in 10.14.0-rc.1 but working fine in 10.13.4

Using latest richdocuments-4.2.0 in both cases.

* Navigate to the Documents folder, click on Example.odt
* the document overview page of richtdocuments opens. BAD
* It tries to load something, but the spinner never finished. BAD
![grafik](https://github.com/owncloud/core/assets/1108546/ed61f8eb-08c9-4005-b7d7-308321a5b2c9)

----

Expected behaviour:
* as seen when swapping 10.14.0-rc.1 with released 10.13.4

![grafik](https://github.com/owncloud/core/assets/1108546/1697c3e9-5832-4e4b-9dc8-ff8fa05b7c91)

**Comment by jnweiger** [comment]
Serverlog seems normal to me:
```
{"reqId":"ZdTnAOeuXPSBskVVMdEObQAAAAI","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/apps\/richdocuments\/documents.php\/index?fileId=14&dir=%2FDocuments","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnAOeuXPSBskVVMdEObQAAAAI","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/apps\/richdocuments\/documents.php\/index?fileId=14&dir=%2FDocuments","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnAOeuXPSBskVVMdEObQAAAAI","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/apps\/richdocuments\/documents.php\/index?fileId=14&dir=%2FDocuments","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnAOeuXPSBskVVMdEObQAAAAI","level":1,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"richdocuments","method":"GET","url":"\/apps\/richdocuments\/documents.php\/index?fileId=14&dir=%2FDocuments","message":"Generating WOPI Token for file 14, version 0."}
{"reqId":"ZdTnAOeuXPSBskVVMdEObQAAAAI","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"richdocuments","method":"GET","url":"\/apps\/richdocuments\/documents.php\/index?fileId=14&dir=%2FDocuments","message":"discovery.xml found in cache"}
{"reqId":"ZdTnAOeuXPSBskVVMdEObQAAAAI","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"richdocuments","method":"GET","url":"\/apps\/richdocuments\/documents.php\/index?fileId=14&dir=%2FDocuments","message":"generateFileToken(): Issuing token, editor: admin, file: 14, version: 0, owner: admin, token: AyHTGkkZ0btTv8CQydH0ap1gHVc3Xjat"}
{"reqId":"ZdTnAOeuXPSBskVVMdEObQAAAAI","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"richdocuments","method":"GET","url":"\/apps\/richdocuments\/documents.php\/index?fileId=14&dir=%2FDocuments","message":"Issued token: {\"status\":\"success\",\"access_token\":\"AyHTGkkZ0btTv8CQydH0ap1gHVc3Xjat\",\"access_token_ttl\":1708487584000,\"sessionid\":\"0\"}"}
{"reqId":"ZdTnAOeuXPSBskVVMdEObQAAAAI","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"richdocuments","method":"GET","url":"\/apps\/richdocuments\/documents.php\/index?fileId=14&dir=%2FDocuments","message":"discovery.xml found in cache"}
{"reqId":"ZdTnAK5khZl6K3Qz2zH9rwAAAAA","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/core\/js\/oc.js?v=5229f3a7a435017456008bd1fd476f35","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnAK5khZl6K3Qz2zH9rwAAAAA","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/core\/js\/oc.js?v=5229f3a7a435017456008bd1fd476f35","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnAK5khZl6K3Qz2zH9rwAAAAA","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/core\/js\/oc.js?v=5229f3a7a435017456008bd1fd476f35","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnAK5khZl6K3Qz2zH9rwAAAAA","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/core\/js\/oc.js?v=5229f3a7a435017456008bd1fd476f35","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnAKuXUsqBMO8de9V57AAAAAQ","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnAKuXUsqBMO8de9V57AAAAAQ","level":0,"time":"2024-02-20T17:53:04+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnA5wwAYnhMm920iZIzgAAAAM","level":0,"time":"2024-02-20T17:53:07+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","message":"token 5c21ed0e314229946a24907fa23a0d7c8e086db3e37abddf780b902f23fcc5114a0dea653235f8ecaf6f9994540d94fee0f3e30146bc407f84d5bfe7a0cda759 with token id 3 found, validating"}
{"reqId":"ZdTnA5wwAYnhMm920iZIzgAAAAM","level":0,"time":"2024-02-20T17:53:07+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\Authentication\\Token\\DefaultTokenProvider::updateTokenActivity","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","message":"updating activity of token 3 to 1708451587"}
{"reqId":"ZdTnA5wwAYnhMm920iZIzgAAAAM","level":0,"time":"2024-02-20T17:53:07+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","message":"token 5c21ed0e314229946a24907fa23a0d7c8e086db3e37abddf780b902f23fcc5114a0dea653235f8ecaf6f9994540d94fee0f3e30146bc407f84d5bfe7a0cda759 with token id 3 found, validating"}
{"reqId":"ZdTnD1q7PkeLjweoOWq2IwAAAAY","level":0,"time":"2024-02-20T17:53:19+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
{"reqId":"ZdTnD1q7PkeLjweoOWq2IwAAAAY","level":0,"time":"2024-02-20T17:53:19+00:00","remoteAddr":"80.139.80.57","user":"admin","app":"OC\\User\\Session::validateToken","method":"GET","url":"\/ocs\/v2.php\/apps\/notifications\/api\/v1\/notifications?format=json","message":"token 89c0ba7f0c0af3380d055c92f41a2ece26c24c7b053c97a8188672403196c70ff76fe0cb23a272761325f4dfeb1d27a19b808c0ee0965baaed9030e2bfe2f0e7 with token id 5 found, validating"}
```

**Comment by DeepDiver1975** [comment]
Oh Lord. They are still using tipsy .......

Will have a look tomorrow

**Comment by DeepDiver1975** [comment]
https://github.com/owncloud/richdocuments/pull/531 - this pr should have been merged some time ago.

**Comment by pako81** [comment]
Confirmed fixed on 10.14.0-RC1 with https://github.com/owncloud/richdocuments/pull/531. Related to https://github.com/owncloud/core/pull/41099.

Maybe also Custom Groups broken with 10.14.0-RC1 as https://github.com/owncloud/customgroups/pull/610 was not included in 0.8.0?

**Comment by jnweiger** [comment]
If possible, I'd like to avoid a richdcuments release now.
Can we revert/postpone the core change and re-allow tipsy at least for this release please?

**Comment by phil-davis** [comment]
Workaround fix is #41188
